client/consensus/babe/src/tests.rs

```diff
@@ -43,7 +43,6 @@ use sp_runtime::{
 	generic::{Digest, DigestItem},
 	traits::Block as BlockT,
 };
-use sp_timestamp::InherentDataProvider as TimestampInherentDataProvider;
```

Comment:
Same question? Can we remove the crate?

Reply:
No here not. In the Babe and Aura tests the slots are computed using the systemtime